lib/mint/web_socket/frame.ex

```diff
@@ -431,6 +431,10 @@ defmodule Mint.WebSocket.Frame do
     {:close, code, reason}
   end
 
+  def translate({:continuation, _, _, _, _}) do
```

Comment:
minor style nit, we can use the record syntax here

```suggestion
  def translate(continuation()) do
```